server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/sql/TestSqlHostVariable.java

```diff
@@ -279,11 +279,11 @@ public class TestSqlHostVariable {
           + "       01 {$*VAR-NAME1}.\n"
           + "       04 {$*VAR`->VAR`->VAR-NAME-LENGTH`->VAR-DATA-LENGTH`->VAR-FILE-OPTION`->VAR-NAME} USAGE IS SQL TYPE IS XML AS CLOB-FILE OCCURS {123456|1} TIMES.\n"
           + "        PROCEDURE DIVISION.\n"
-          + "           DISPLAY {$VAR}(1).\n"
-          + "           DISPLAY {$VAR-NAME-LENGTH}(1).\n"
-          + "           DISPLAY {$VAR-DATA-LENGTH}(1).\n"
-          + "           DISPLAY {$VAR-FILE-OPTION}(1).\n"
-          + "           DISPLAY {$VAR-NAME}(1).";
+          + "           DISPLAY {$VAR}.\n"
```

Comment:
The `(1)` should be there, the variables are arrays.

Reply:
updated